sdk/python/feast/infra/offline_stores/snowflake_source.py

```diff
@@ -191,8 +191,10 @@ def validate(self, config: RepoConfig):
 
     def get_table_query_string(self) -> str:
         """Returns a string that can directly be used to reference this table in SQL."""
-        if self.database and self.table:
+        if self.database and self.schema and self.table:
             return f'"{self.database}"."{self.schema}"."{self.table}"'
+        elif self.schema and self.table:
+            return f'"{self.schema}.{self.table}"'
```

Comment:
probably doesn't matter, but should this be `"{self.schema}"."{self.table}"`? Just inferring from the line above...

Reply:
Yes you're right. Forgot to push that change. Pushed it just now!